components/Alert/AlertProvider.jsx

```diff
@@ -0,0 +1,49 @@
+import { createContext, useState, useRef, useEffect } from "react";
+
+const initialState = {
+    text: "",
+    color: "",
+    icon: "",
+    totalAlerts: 0,
+};
+
+export const AlertContext = createContext({ ...initialState, setAlert: () => {} });
```

Comment:
why does setAlert have to be here but not resetAlert?

Reply:
It doesn't I don't think lol

Reply:
Actually just looked, these are default values in case the context cannot be accessed properly, think it's good practice to add all function in here for some reason?

Reply:
Looked more, ppl go either way with nothing or everything inside. Think nothing looks cleaner so will change all to nothing